bin/cmd.js

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env node
 
-const opts = require('../options.js')
-require('standard-engine').cli(opts)
+import('standard-engine').then(function (semiStandardEngine) {
+  import('../options.js').then(function (options) {
```

Comment:
These can be static imports rather than dynamic ones, right?

```javascript
import { cli } from 'standard-engine';
import options from '../options.js';
```

Reply:
could try... don't know if cli/bin runs as type module...

Reply:
Yeah, or we should copy what we do in `standard`:

```js
#!/usr/bin/env node
/* eslint-disable no-var, no-eval */

var match = process.version.match(/v(\d+)\.(\d+)/)
var major = parseInt(match[1], 10)
var minor = parseInt(match[2], 10)

if (major >= 12 || (major === 12 && minor >= 20)) {
  eval('import("standard-engine")').then(function (standardEngine) {
    eval('import("../options.js")').then(function (options) {
      standardEngine.cli(options.default)
    })
  })
} else {
  console.error('semistandard: Node 12.20.0 or greater is required. `semistandard` did not run.')
}
```

Reply:
Is the `eval('import("standard-engine")')` there really necessary @LinusU? 🤔

Reply:
The suggestion by @LinusU is better, that way people who are running CI on non-ESM versions can still have their tests pass

Reply:
If we copy, what we do in `standard`, we should include an "Old test" to ensure that the CLI is exiting successfully, see: https://github.com/standard/standard/blob/master/.github/workflows/old-test.yml.

Reply:
> Is the `eval('import("standard-engine")')` there really necessary @LinusU? 🤔

Yes it is necessary to run on older Node.js versions without errors.

Reply:
should i copy over the hole old-test.yml file?
i changed the code so it looks the same way as standard

Reply:
> should i copy over the hole old-test.yml file?
> i changed the code so it looks the same way as standard

Yes, I guess. :smile:

Reply:
I wonder if it wouldn't be better if you do a try catch instead... what if you run node 14.0.0 that don't have import yet?

```js
// this only check if it is above 12.20 or higher
if (major >= 12 || (major === 12 && minor >= 20)) {
```

something like: 

```js
try {
  eval('import("standard-engine")').then(function (standardEngine) {
    eval('import("../options.js")').then(function (options) {
      standardEngine.cli(options.default)
    })
  })
} catch (err) {
  console.error('semistandard: Node 12.20.0 or greater is required. `semistandard` did not run.')
}
```

Reply:
at least it should not be able to install it if we have `node.engines` anyway?